### DISCUSSION TITLE
Shared signals not appearing

### DISCUSSION THREAD

**Discussion by Tthongsy** [datalust/seq-tickets#1281] [Help] [answered]
Hi,
Sorry for the double post, I had another issue where shared signals were not appearing. I have the signals set to share and they aren't appearing in other Seq instances. Is there anything I can do or I might have missed?

Thanks,
Tommy T

**Comment by nblumhardt** [comment, marked as answer]
Thanks for your message. Sharing a signal will only share it with other users on the same Seq instance. If you want to move signals between instances, using `seqcli signal list --json` and `seqcli signal import` at the command-line can do this.

**Reply by Tthongsy** [reply]
Hi,
So shared signals aren't appearing for other users on the same Seq instance is what I mean't sorry. I wasn't sure if I missed any other steps besides setting the signal to 'Share'.

Thanks,
Tommy T

**Reply by nblumhardt** [reply]
One other thing to check - by default, each user has their own workspace, and shared items are only included when added to it.

In the signal bar, other users will be able to find your shared signals by searching ("Find signals and queries")

or

In the workspace selector (will say "Personal", in the top navbar beside the Seq logo), other users can choose the "x" to close the workspace and see all shared items by default.

Let me know if this solves it! :-)

Best regards,
Nick
